--- com.redhat.lvm2.c.orig	2014-01-09 13:41:08.897353979 +0200
+++ com.redhat.lvm2.c	2014-01-09 13:41:38.934491256 +0200
@@ -10608,7 +10608,9 @@
 static void
 lvm_object_notify (GDBusObject *object, GDBusInterface *interface)
 {
-  g_object_notify (G_OBJECT (object), ((_ExtendedGDBusInterfaceInfo *) g_dbus_interface_get_info (interface))->hyphen_name);
+  _ExtendedGDBusInterfaceInfo *info = (_ExtendedGDBusInterfaceInfo *) g_dbus_interface_get_info (interface);
+  if (info)
+    g_object_notify (G_OBJECT (object), info->hyphen_name);
 }
 
 /**
